plugins/auth/login.js

```diff
@@ -48,12 +48,9 @@ module.exports = config => ({
                     if (!model) {
                         return factory.create({
                             username,
-                            token: githubToken,
-                            password: config.password
```

Comment:
_note: password hasn't been required here since we setup password on the UserFactory in `bin/server.js`_